inc/scroom/layeroperations.hh

```diff
@@ -29,6 +30,42 @@ public:
                     Scroom::Utils::Stuff cache);
 };
 
+class PipetteCommonOperations : public CommonOperations, public PipetteLayerOperations
```

Comment:
Not sure what the purpose of this empty class is?

Reply:
The reason for it was to 'group' the two `PipetteCommonOperationsRGB` and `PipetteCommonOperationsCMYK` when creating the `Ptr`'s of specific `LayerOperations` . If we don't want this class anymore, we would need to introduce  `typedef boost::shared_ptr<PipetteCommonOperationsRGB> Ptr;` (and similarly for CMYK). Then we can remove `PipetteCommonOperations`. Will be fixed